packages/recs/src/Component.ts

```diff
@@ -75,7 +75,24 @@ export function setComponent<S extends Schema, T = undefined>(
 ) {
   const prevValue = getComponentValue(component, entity);
   for (const [key, val] of Object.entries(value)) {
-    component.values[key].set(entity, val);
+    if (component.values[key]) {
+      component.values[key].set(entity, val);
+    } else if (!component.metadata?.tableId) {
+      // If component metadata includes the tableId (from `defineStoreComponents`),
+      // we can ignore this value without logging anything.
```

Comment:
Why is this? Because the update includes additional numeric indices as keys but they are not part of the component?

Reply:
Correct!